ytfzf

```diff
@@ -941,8 +942,7 @@ scrape_subscriptions () {
 	done << EOF
 $( sed \
 	-e "s/#.*//" \
-	-e "/^[[:space:]]*$/d" \
-	-e "s/[[:space:]]*$//" \
+	-e "s/[[:space:]]*//g" \
```

Comment:
we could remove the lines which are empty
`/^[[:space:]]*$/d`
else the while read loop will take an empty line

Reply:
or maybe add a check to see if $url in empty before scrape_channel